enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -431,6 +431,17 @@ const theme = {
         backgroundColor: "#95A5A6",
       },
     },
+    
+    collectionBrowser: {
+       breadcrumbs: {
+         expandButton: {
+           backgroundColor: "yellow";
+           hoverBackgroundColor: "red";
+           textColor: "blue";
+           hoverTextColor: "green";
```

Comment:
You might want to use hex colors here instead, as color names doesn't work in all cases (it's a known limitation)